zstacklib/zstacklib/gpu/vendors/nvidia.py

```
@@ -312,6 +312,119 @@ def _parse_vgpu_output(output):
+                memory_utilization=mem_pct,
+            ))
+        return result
+
+    @classmethod
+    def _parse_pmon_output(cls):
+        """Call nvidia-smi pmon and return {pid: (sm_util, fb_mib)}.
+
+        sm_util: GPU SM utilization percentage from pmon.
+        fb_mib: framebuffer usage in MiB (absolute value, converted to % later using worker.allocatedMemoryMb).
+        """
+        r, o, e = bash_roe("nvidia-smi pmon -c 1 -s mu")
```

Ruff (0.15.11)

[warning] Unpacked variable `e` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)